api/base.py

```
@@ -890,37 +904,51 @@ def fetch_response():
+            logger.error(f"获取章节检测题目失败, 达到最大重试次数: {e}")
             return StudyResult.ERROR
 
         _ORIGIN_HTML_CONTENT = final_resp.text  # 用于配合输出网页源码, 帮助修复#391错误
 
         # 搜题
         total_questions = len(questions["questions"])
         found_answers = 0
-        for q in questions["questions"]:
+        query_delay = self.kwargs.get("query_delay", 0)
+        answers = self.tiku.query_all(questions["questions"], query_delay=query_delay)
+        for q, res in zip(questions["questions"], answers):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)